// fetch("https://api.chucknorris.io/jokes/random")
//   .then((res) => res.json())
//   .then((res) => {
//     throw new Error("error"); //zorgt ervoor dat er een errorboodschap verschijnt (geforceerd hier)
//   })
//   .catch((err) => alert("An error occured")); //zorgt ervoor dat er een error verschijnt als er iets foutloopt

// async function fetchChuckJoke()
